CONTRIBUTING.md

```diff
@@ -5,14 +5,16 @@ We love pull requests and issues, they're our favorite.
 However, before submitting, please review the following:
 
 For bug reports:
-  - Please make sure the bug is reproducible, and give us the steps to reproduce it, so that we can dig into the problem.
```

Comment:
i'm confused why this is an improvement; it seems like it'd render the same in the markdown, but it's much less readable in raw form.

Reply:
No, i don’t agree with that rationale - i don’t care about the editing experience, but the reading experience in raw form is important to me.

Reply:
okay, I'll rollback this one, how about the others? Look good to you?

Reply:
i think so; i'll take another look once the diff is updated

Reply:
I have a concern here, some of them, including the table of contents, start from the beginning of the lines, won't this make some inconsistency? and in that case of table of contents, it looks different.

Let me know if I should just "rollback" or the consistency should also be maintained, thanks.

Reply:
Okay, I'll just rollback those changes.